packages/cms-lib/lib/config.js

```diff
@@ -353,6 +377,118 @@ const deleteEmptyConfigFile = () => {
   );
 };
 
+const getConfigVariablesFromEnv = () => {
+  const env = process.env;
+
+  return {
+    apiKey: env[HUBSPOT_API_KEY],
+    clientId: env[HUBSPOT_CLIENT_ID],
+    clientSecret: env[HUBSPOT_CLIENT_SECRET],
+    personalAccessKey: env[HUBSPOT_PERSONAL_ACCESS_KEY],
+    portalId: parseInt(env[HUBSPOT_PORTAL_ID], 10),
+    refreshToken: env[HUBSPOT_REFRESH_TOKEN],
+  };
+};
+
+const generatePersonalAccessKeyPortalConfig = (portalId, personalAccessKey) => {
```

Comment:
The naming of these functions is a bit confusing because it says "PortalConfig` which elsewhere only refers to the configuration for an individual portal.

Reply:
Renamed these